corehq/apps/users/models.py

```diff
@@ -2991,6 +2991,48 @@ class Meta(object):
         unique_together = ('domain', 'user_id', 'app_id')
 
 
+class UserOptionToggles(models.Model):
+    """
+    Additional option toggles for a user
+    """
+    user = models.OneToOneField(
+        User,
+        related_name='option_toggles',
+        on_delete=models.CASCADE,
+    )
```

Comment:
Do you know if this is a practice we've considered acceptable?  Referencing a representation of the user in SQL would be _really_ useful (I want a `UserLocation` class that links those two models for easier querying than is possible in couch/ES).  However, I don't want to introduce confusion around when we're dealing with a couch user vs a django user.

Reply:
We do it [for API keys here](https://github.com/dimagi/commcare-hq/blob/582526c6865a0679359135a9a3b2b75b1ec758a9/corehq/apps/users/models.py#L3044).

You make a good point: "user" is very ambiguous, and this should have been `django_user`.

But after considering y'all's feedback, I've decided not to create this model.